libpromises/generic_agent.h

```diff
@@ -77,8 +77,8 @@ typedef struct
             unsigned int parser_warnings;
             unsigned int parser_warnings_error;
             bool eval_functions;
-            bool show_classes;
-            bool show_variables;
+            char* show_classes;
+            char* show_variables;
```

Comment:
A style linter would be really nice to save me time for this kind of small fix :)